packages/js/src/generators/library/library.spec.ts

```diff
@@ -731,251 +732,303 @@ describe('lib', () => {
       const readme = tree.read('libs/my-lib/README.md', 'utf-8');
       expect(readme).toContain('nx test my-lib');
     });
+  });
 
-    describe('--buildable', () => {
-      it('should generate the build target', async () => {
-        await libraryGenerator(tree, {
-          ...defaultOptions,
-          name: 'myLib',
-          buildable: true,
-          compiler: 'tsc',
-        });
-
-        const config = readProjectConfiguration(tree, 'my-lib');
-        expect(config.targets.build).toEqual({
-          executor: '@nrwl/js:tsc',
-          options: {
-            assets: ['libs/my-lib/*.md'],
-            main: 'libs/my-lib/src/index.ts',
-            outputPath: 'dist/libs/my-lib',
-            tsConfig: 'libs/my-lib/tsconfig.lib.json',
-          },
-          outputs: ['{options.outputPath}'],
-        });
+  describe('--buildable', () => {
```

Comment:
The following test are not changed. The `desribe('--buildable', () => {` block was nested into the ` describe('--unit-test-runner jest', () => {` which was wrong I think. I removed the nesting.